### PR TITLE
Hide spinners for type="number"

### PR DESCRIPTION
**Pull request by dangodev**
Hides these little goobers:

Screen Shot 2020-04-24 at 13 58 35

### CODE REVIEW

File: src/components/_input.scss [hunk 1]
```diff
@@ -42,6 +42,7 @@
     transition-duration: 150ms;
     transition-property: border-color;
     appearance: none;
+    -moz-appearance: textfield; // stylelint-disable-line
```
Comment (davidleger95) [inline src/components/_input.scss:45]: why is this needed?

Comment (dangodev) [reply src/components/_input.scss:45]: Firefox doesn’t respect the `::webkit-inner-spin-button` 🤷

Comment (dangodev) [reply src/components/_input.scss:45]: And setting it to `-moz-appearance: none` still shows it too 🤷 🤷‍♂️ 🤷‍♀️
